### DISCUSSION TITLE
ENV segments not working in WSL/Linux/Ubuntu?

### DISCUSSION THREAD

**Discussion by brwilkinson** [JanDeDobbeleer/oh-my-posh#1480] [Troubleshoot] [answered]
Anyone have any ideas on why my ENV vars are not working in Linux?

below is linux versus PowerShell, both have the same ENV Vars set.

![image](https://user-images.githubusercontent.com/3605266/147014176-4a2c5bde-8040-49c6-aa78-5d8f2354a3be.png)

Both are pwsh.

E.g. config

```json
                {
                    "type": "envvar",
                    "style": "plain",
                    "powerline_symbol": "\uE0B0",
                    "foreground": "#000000",
                    "background": "#808000",
                    "properties": {
                        "var_name": "AzAccount"
                    }
                },
```

**Comment by JanDeDobbeleer** [comment, marked as answer]
Can you paste the output of `Write-PoshDebug`?

**Reply by brwilkinson** [reply]
Sure, did you see the images? Or did you want a text view?

**Reply by JanDeDobbeleer** [reply]
Ah, I see you're using an older version, can you update and reload the shell ? That way you'll also see what oh-my-posh is doing.

**Reply by brwilkinson** [reply]
ooh nice. thanks.

You know I realized it was just a case-sensitive issue that I missed before, I thought I had double checked, but I missed a char.

Thanks for the assist ✅

**Reply by brwilkinson** [reply]
Thanks for you work on this module by the way, I love it.
